### PR TITLE
lava-action: fix no vulnerability_count issue

### PR DESCRIPTION
**Pull request by jesusfcr**
This PR fixes an issue when no vulnerabilities were detected and add a test for that case.

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -214,3 +216,47 @@ jobs:
         run: |
           echo "::error::unexpected status code: got: ${{ steps.lava.outputs.status }}, want: ${{ env.WANT_STATUS }}"
           exit 1
+
+  test-no-vulns:
+    env:
+      WANT_STATUS: 0 # ExitCodeInfo
```
Comment (jroimartin) [inline .github/workflows/test.yaml:222]: [ExitCodeInfo](https://github.com/adevinta/lava/blob/a7eabad5fcf9c781bc2dc296606eea121917b69f/internal/report/report.go#L309) is `100`.

```suggestion
      WANT_STATUS: 0 # No vulnerabilities found
```

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -214,3 +216,47 @@ jobs:
...
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Run Lava Action
+        id: lava
+        uses: ./
+        with:
+          comment-pr: 'true'
+        env:
+          TEST_WORKDIR: .github
```
Comment (jroimartin) [inline .github/workflows/test.yaml:234]: Let's keep it like this for now. But we should create directories under `/testdata` for the different test cases. So, it is more obvious.

**Review by jroimartin — CHANGES_REQUESTED** [review]
Just a minor comment.

**Comment by jroimartin** [comment]
Also, please, add a short description to the pull request.

File: .github/workflows/test.yaml (hunk 1 above)
Comment (jesusfcr) [reply .github/workflows/test.yaml]: done

**Review by jroimartin — APPROVED** [review]
LGTM. Merge and tag as `v0.3.1`.

**Pull request merged**